arlo-client/src/components/Audit/Progress/index.tsx

```diff
@@ -20,13 +22,28 @@ interface IProps {
 const Progress: React.FC<IProps> = ({ refreshId }: IProps) => {
   const { electionId } = useParams<{ electionId: string }>()
   const jurisdictions = useJurisdictions(electionId, refreshId)
+  const [isDetailOpen, setIsDetailOpen] = useState(false)
+  const [jurisdictionDetail, setJurisdictionDetail] = useState(jurisdictions[0])
```

Comment:
If you wanted, you could combine these into one state value of type `IJurisdiction | null` which would either be the jurisdiction to show the detail for or `null` when the detail modal is closed.

Reply:
@jonahkagan :thinking: That sounds neat. I'll try that. I think it could fix your first comment as well.